packages/server/src/api/officeHoursRoutes.ts

```diff
@@ -1,27 +1,99 @@
-import { User, Course } from "@template/common";
-import { ServerRoute } from "@hapi/hapi";
+import {
+  GetProfileResponse,
+  GetCourseResponse,
+  TAUpdateStatusParams,
+  TAUpdateStatusResponse,
+  CreateQuestionParams,
+  CreateQuestionResponse,
+  UpdateQuestionParams,
+} from "@template/common";
+import { ServerRoute, ResponseObject } from "@hapi/hapi";
 import Joi from "@hapi/joi";
 
 import { MOCK_GET_PROFILE_RESPONSE } from "../mocks/getProfile";
 import { MOCK_GET_COURSE_RESPONSE } from "../mocks/getCourse";
+import {
+  MOCK_TA_UPDATE_STATUS_ARRIVED_RESPONSE,
+  MOCK_TA_UPDATE_STATUS_DEPARTED_RESPONSE,
+} from "../mocks/taUpdateStatus";
+import { MOCK_CREATE_QUESTION_RESPONSE } from "../mocks/createQuestion";
+import {
+  MOCK_STUDENT_UPDATE_QUESTION_RESPONSE,
+  MOCK_TA_UPDATE_QUESTION,
+} from "../mocks/updateQuestion";
 
 export const officeHoursRoutes: ServerRoute[] = [
   {
     method: "GET",
     path: "/v1/profile",
-    handler: async (request, h): Promise<User> => {
+    handler: async (request, h): Promise<GetProfileResponse> => {
       return MOCK_GET_PROFILE_RESPONSE;
     },
   },
   {
     method: "GET",
     path: "/v1/courses/{course_id}",
-    handler: async (request, h): Promise<Course | string> => {
+    handler: async (
+      request,
+      h
+    ): Promise<GetCourseResponse | ResponseObject> => {
       if (request.params.course_id === "169") {
+        // coures_id 169 = Fundies 1
         return MOCK_GET_COURSE_RESPONSE;
       } else {
-        // todo: we should fix this later
-        return h.response("The course did not exist").code(404) as any;
+        return h.response("The course did not exist").code(404);
+      }
+    },
+  },
+  {
+    method: "GET",
+    path: "/v1/courses/{course_id}/ta/change_status",
```

Comment:
According to the [api docs](https://www.notion.so/API-Docs-aa0846685baf4f8ca5f187cd2c6ef38d), this should be a PATCH, instead of a GET.

Reply:
Whoops! Thanks for the comment. Just got that changed